src/app/clusters/scenes/SceneTableImpl.cpp

```diff
@@ -350,28 +535,85 @@ void DefaultSceneTableImpl::Finish()
     mSceneEntryIterators.ReleaseAll();
 }
 
-CHIP_ERROR DefaultSceneTableImpl::SetSceneTableEntry(FabricIndex fabric_index, const SceneTableEntry & entry)
+CHIP_ERROR DefaultSceneTableImpl::GetGlobalSceneCount(uint8_t & scene_count)
+{
+    GlobalSceneCount global_count;
+
+    ReturnErrorOnFailure(global_count.Load(mStorage));
```

Comment:
Like here. if not found, assign 0 to `scene_count`

Reply:
Put the 0 in the global_count.Load() instead.